app/plugins/openwhisk.md

```diff
@@ -76,9 +76,10 @@ Attribute                                | Description
 `config.service_token`<br>*optional*     | The service token to access Openwhisk resources.
 `config.https_verify`<br>*optional*      | Set it to true to authenticate Openwhisk server. Defaults to `false`.
 `config.https`<br>*optional*             | Use of HTTPS to connect with the OpenWhisk server. Defaults to `true`.
-`config.timeout`<br>*optional*           | Timeout in seconds before aborting a connection to OpenWhisk server. Defaults to `60 seconds`.
-`config.result`<br>*optional*            | Return only the result of the Action invoked. Defaults to `true`.
-`config.keepalive`<br>*optional*         | Defines for how long an idle connection to OpenWhisk server will live before being closed. Defaults to `60 seconds`.
+`config.timeout`<br>*optional*           | Timeout in millisecond before aborting a connection to OpenWhisk server. Defaults to `60000`.
```

Comment:
Or maybe you wanted to highlight that you found a typo in the word (plural form?), in which case, sorry!